pkg/controller/scyllacluster/resource.go

```diff
@@ -896,14 +971,19 @@ func MakeAgentAuthTokenSecret(c *scyllav1.ScyllaCluster, authToken string) (*cor
 		return nil, err
 	}
 
+	labels := map[string]string{}
+	maps.Copy(labels, c.Labels)
+	maps.Copy(labels, naming.ClusterLabels(c))
+
 	return &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      naming.AgentAuthTokenSecretName(c.Name),
 			Namespace: c.Namespace,
 			OwnerReferences: []metav1.OwnerReference{
 				*metav1.NewControllerRef(c, scyllaClusterControllerGVK),
 			},
-			Labels: naming.ClusterLabels(c),
+			Labels:      labels,
+			Annotations: maps.Clone(c.Annotations),
```

Comment:
nit: lets make it consistent that labels and annotations are defined in variables and these variables are used in spec

Reply:
tbh, I am not seeing the value in that as it add extra jump / dereference for the reader and doesn't use the value + it' not extensible as the `Clone` and empty map pattern differ anyways.

Reply:
I meant to be consistent with style within single commit. In other places (MakeServiceAccount, MakeRoleBinding) there's an extra variable defined - which I like more because adding anything new values requires shorter diff - and in other there's explicit clone.